orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt

```diff
@@ -87,26 +90,30 @@ object RunTaskHandlerTest : SubjectSpek<RunTaskHandler>({
   val repository: ExecutionRepository = mock()
   val stageNavigator: StageNavigator = mock()
   val task: DummyTask = mock()
+  val cloudProviderAwareTask: DummyCloudProviderAwareTask = mock()
   val timeoutOverrideTask: DummyTimeoutOverrideTask = mock()
   val exceptionHandler: ExceptionHandler = mock()
   val clock = fixedClock()
   val contextParameterProcessor = ContextParameterProcessor()
+  val dynamicConfigService: DynamicConfigService = mock()
+  val taskExecutionInterceptors: List<TaskExecutionInterceptor> = listOf(mock())
 
   subject(GROUP) {
     RunTaskHandler(
       queue,
       repository,
       stageNavigator,
       contextParameterProcessor,
-      listOf(task, timeoutOverrideTask),
+      listOf(task, timeoutOverrideTask, cloudProviderAwareTask),
       clock,
       listOf(exceptionHandler),
-      emptyList(),
-      NoopRegistry()
+      taskExecutionInterceptors,
```

Comment:
I removed the `emptyList()` so I could test the max back off set in the `taskExecutionInterceptors`.  This meant I had to update a fair amount of these tests.